CMIP6_license.json

```diff
@@ -1,15 +1,33 @@
 {
-    "license":[
-        "CMIP6 model data produced by <Your Centre Name> is licensed under a Creative Commons Attribution-[NonCommercial-]ShareAlike 4.0 International License (https://creativecommons.org/licenses). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file)[ and at <some URL maintained by modeling group>]. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law."
-    ],
+    "license":{
```

Comment:
Are you happy that this form is easily translatable into a pattern for the license in with the MIP tables? https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_CV.json#L2723

Reply:
Not sure. I think so, but maybe we should check with Chris.

Reply:
I have tweaked this slightly, so that a simple string replace would work

Reply:
I note, as this is merged @mauzey1 will needs a heads up about the changes - so that the cmip6-cmor-table and CMOR updates can happen without a problem